fpm/recipes/gof3r/recipe.rb

```diff
@@ -0,0 +1,21 @@
+class Terraform < FPM::Cookery::Recipe
```

Comment:
Updated to fix and also make more consistent with the way we've changed the terraform manifest to look.